src/cregistry/registry.c

```diff
@@ -258,6 +259,8 @@ int reg_attach(reg_registry* reg, const char* path, reg_error* errPtr) {
                                     sizeof(sqlite_int64)/sizeof(int));
                             Tcl_InitHashTable(&reg->open_files,
                                     TCL_STRING_KEYS);
+                            Tcl_InitHashTable(&reg->open_distfiles,
+                                    TCL_STRING_KEYS);
```

Comment:
If you decide to switch to rowid keys, remember to also changes this to `sizeof(sqlite_int64) / sizeof(int)`.